### ISSUE TITLE
Error when specifying nonexistent environment

### ISSUE THREAD

**Issue by jakubknejzlik** [cdrx/rancher-gitlab-deploy#15]
I've run into small issue when I accidentally specified wrong environment name which triggers this error:
```
upgrade --environment env --service betag-wb-frontend
Traceback (most recent call last):
  File "/usr/local/bin/upgrade", line 11, in <module>
    load_entry_point('rancher-gitlab-deploy==1.2', 'console_scripts', 'rancher-gitlab-deploy')()
  File "/usr/local/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/rancher_gitlab_deploy-1.2-py2.7.egg/rancher_gitlab_deploy/cli.py", line 68, in main
    if not environment_id:
UnboundLocalError: local variable 'environment_id' referenced before assignment
```

Would be great to get same feedback as for nonexistent service:
```
Error: Unable to find a service called 'betag-wb-frontend', does it exist in Rancher?
```

**Comment by cdrx** [comment]
Thanks, I'll fix that

**Comment by cdrx** [comment]
Fixed in 542ab54
